docs/sources/getting-started/getting-started-prometheus.md

```diff
@@ -0,0 +1,69 @@
++++
+title = "Getting started with Grafana and Prometheus"
+description = "Guide for getting started with Grafana"
+keywords = ["grafana", "intro", "guide", "started"]
+type = "docs"
+aliases = ["/docs/grafana/latest/guides/gettingstarted","/docs/grafana/latest/guides/getting_started"]
+[menu.docs]
+identifier = "getting_started-grafana-prometheus"
+parent = "guides"
+weight = 300
++++
+
+# Getting Started with Grafana and Prometheus
```

Comment:
Should this be "_Get_ started ..."?

Reply:
We do not use gerunds when talking about tasks. For example, "Add data source", which is probably not right, you are "explaining" how to do a "task".

In this case, the section is about "Getting started", so this should stay.

Reply:
Gotcha. I thought this was considered a task topic, hence my comment. I believe it should be in sentence case though.

Reply:
Correct. All headings should be in sentence case.

Reply:
Diana and I decided to go with Getting Started > With Grafana and Prometheus